fixieai/client/agent.py

```diff
@@ -102,6 +103,42 @@ def owner(self) -> Optional[str]:
         assert isinstance(owner_username, str)
         return owner_username
 
+    @property
+    def query_url(self) -> Optional[str]:
+        """Return the query URL for this Agent."""
+        if self._metadata is None:
+            return None
+        url = self._metadata["owner"]["query_url"]
```

Comment:
From code inspection I think `queryUrl` is unused? In https://github.com/fixie-ai/fixie-sdk/pull/50 I only wired up `funcUrl` for that reason